run.go

```diff
@@ -53,49 +59,56 @@ func init() {
 	cmdRun.Flag.Var(&excludedPaths, "e", "Excluded paths[].")
 	cmdRun.Flag.BoolVar(&vendorWatch, "vendor", false, "Watch vendor folder")
 	cmdRun.Flag.StringVar(&buildTags, "tags", "", "Build tags (https://golang.org/pkg/go/build/)")
+	exit = make(chan bool)
 }
 
-var appname string
-
 func runApp(cmd *Command, args []string) int {
 	ShowShortVersionBanner()
 
-	exit := make(chan bool)
-	crupath, _ := os.Getwd()
+	gps := GetGOPATHs()
+	if len(gps) == 0 {
+		ColorLog("[ERRO] Fail to start[ %s ]\n", "$GOPATH is not set or empty")
+		os.Exit(2)
+	}
+	gopath := gps[0]
 
 	if len(args) == 0 || args[0] == "watchall" {
-		appname = path.Base(crupath)
+		currpath, _ = os.Getwd()
+		appname = path.Base(currpath)
 		ColorLog("[INFO] Uses '%s' as 'appname'\n", appname)
 	} else {
-		appname = args[0]
+		gopathsrc := path.Join(gopath, "src")
+		currpath = path.Join(gopathsrc, args[0])
+		appname = path.Base(currpath)
+
+		// Check if passed Bee application path/name exists
+		// in $GOPATH/src workspace
+		if !isExist(currpath) {
```

Comment:
do we need to check all gopath? As gopath may include multi paths, I think we need to check all these paths